docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+---
+id: google-iap
+title: Google IAP
+sidebar_label: Google IAP
+---
+
+## Introduction
+
+As of September 2021, Subscription Platform (SubPlat) supports [Google IAP](https://developer.android.com/google/play/billing) integrations for products with Mozilla VPN as one example. An integration allows us to maintain an awareness of a Google IAP subscriber's subscription status for a given android app, including any state changes, which we forward on to the relevant relying party (RP).
```

Comment:
Is`September 2021` correct? I went through the Google IAP Integration and Fast Follows epics but wanted to make sure. Thank you in advance!

Reply:
Thank you! Updated to November 2021.